src/elements/form/Control.ts

```diff
@@ -258,30 +260,42 @@ export class NovoControlElement extends OutsideClick implements OnInit, OnDestro
         event.target.style.height = height;
     }
 
+    checkMaxLength(event) {
```

Comment:
NG2 already does this, not required. Checked out feature branch here and it worked without it.